src/dmd/root/port.d

```diff
@@ -54,6 +54,34 @@ extern (C++) struct Port
         return result;
     }
 
+    /**
+       Compare two slices for equality, in a case-insensitive way
+
+       Params:
+         s1 = string to compare
+         s2 = string to compare
+
+       Returns:
+         `true` if `s1 == s2` regardless of case
+     */
+    extern(D) static int iequals(const char[] s1, const char[] s2)
```

Comment:
* Returns `int` instead of `bool`
* Is if worth mentioning in the documentation that this only works with ASCII (as far as I can tell)?